admin/tools/docker/base/Dockerfile

```diff
@@ -219,11 +217,8 @@ RUN dnf update -y \
        https://apache.jfrog.io/artifactory/arrow/almalinux/$(cut -d: -f5 /etc/system-release-cpe | cut -d. -f1)/apache-arrow-release-latest.rpm \
     && dnf config-manager --set-enabled epel \
     && dnf config-manager --set-enabled powertools \
-    && dnf install -y arrow-devel-$ARROW_VERSION \
-    && dnf install -y arrow-dataset-devel-$ARROW_VERSION \
-    && dnf install -y arrow-flight-devel-$ARROW_VERSION \
-    && dnf install -y gandiva-devel-$ARROW_VERSION \
-    && dnf install -y parquet-devel-$ARROW_VERSION \
+    && dnf install -y arrow11-libs-$ARROW_VERSION \
+    && dnf install -y parquet11-libs-$ARROW_VERSION \
```

Comment:
This change needs to be pushed from the current commit to the very first commit titled "Install arrow".

Reply:
The git history is modified according to the remarks you made